mod-erm-usage-server/src/test/java/org/folio/rest/impl/CounterReportUploadIT.java

```diff
@@ -204,12 +210,42 @@ private void testThatDBIsEmpty() {
     testThatDBSizeIsSize(0);
   }
 
+  private String createReportFromJson(File jsonFile)
+      throws IOException {
+    return createReport(jsonFile, "application/json", false, null);
+  }
+
+  private String createReport(File file, String mimeType, Boolean reportEditedManually,
+      String editReason)
+      throws IOException {
+    String fileAsString = Files.toString(file, StandardCharsets.UTF_8);
+    return createReport(fileAsString, mimeType, reportEditedManually, editReason);
+  }
+
+  private String createReport(String string, String mimeType, Boolean reportEditedManually,
+      String editReason)
+      throws IOException {
+    String fileAsBytes = Base64.getEncoder().encodeToString(string.getBytes());
+    Contents content = new Contents().withData("data:" + mimeType + " ;base64," + fileAsBytes);
+    ReportMetadata metadata = new ReportMetadata().withReportEditedManually(reportEditedManually);
+    if (editReason != null && reportEditedManually) {
+      metadata.setEditReason(editReason);
+    }
+    CounterReportDocument document = new CounterReportDocument()
+        .withReportMetadata(metadata)
+        .withContents(content);
+
+    ObjectMapper objectMapper = new ObjectMapper();
+    return objectMapper.writeValueAsString(document);
+  }
+
```

Comment:
No need for a string representation here.. just return the CounterReportDocument as is. it can then be passed directly to the body() method of RestAssured

```suggestion
private CounterReportDocument createReport(
      String string, String mimeType, Boolean reportEditedManually, String editReason) {
    String fileAsBytes = Base64.getEncoder().encodeToString(string.getBytes());
    Contents content = new Contents().withData("data:" + mimeType + " ;base64," + fileAsBytes);
    ReportMetadata metadata = new ReportMetadata().withReportEditedManually(reportEditedManually);
    if (editReason != null && reportEditedManually) {
      metadata.setEditReason(editReason);
    }
    return new CounterReportDocument().withReportMetadata(metadata).withContents(content);
  }

```